.changes/v3.3.0/680-improvements.md

```diff
@@ -0,0 +1 @@
+* Add support for disk controller type nvme [GH-680]
```

Comment:
Sorry for having this stuck. Could you just merge with master and change .changes to `v3.4.0`. I will just double check after latest merge with master and approve it

Reply:
ping

Reply:
rebased on upstream master and moved .changes to v3.4.0